abci/tests/test_app/app.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
This also looks like it was supposed to be removed in https://github.com/tendermint/tendermint/commit/ea271c534a6955b8abc9e2b5b62bd16a437a81a4 right?